worldedit-core/src/main/java/com/sk89q/worldedit/regions/Region.java

```diff
@@ -365,4 +366,9 @@ default Extent construct(Extent child) {
         }
         return new SingleRegionExtent(child, FaweLimit.MAX, this);
     }
+
+    @Override
+    default ProcessorScope getScope() {
+        return ProcessorScope.REMOVING_BLOCKS;
```

Comment:
Why is `REMOVING_BLOCKS` considered the default? 
I'm not sure if I got the concept behind the scope, but in general, I'd prefer to return a (Enum)Set, as processors might work in multiple scopes (does that make sense?). Then, we wouldn't need a `CUSTOM` constant and could use noneOf or allOf (whatever makes sense) instead.

Reply:
the scope is basically what determines the order in which processors are run. Region processors remove blocks, as they limit blocks to a region. The actual default is CUSTOM, as it's safest to run any processors people add themselves after anything FAWE does, but before FAWE writes history